internal/hud/hud.go

```diff
@@ -174,6 +178,22 @@ func (h *Hud) handleScreenEvent(ctx context.Context, dispatch func(action store.
 				escape()
 			case r == 'R': // hidden key for recovering from printf junk during demos
 				h.r.screen.Sync()
+			case r == 't': // [T]rigger resource update
+				_, selected := h.selectedResource()
+				h.recordInteraction("trigger_resource")
+				payload := []byte(fmt.Sprintf(`{"manifest_names":[%q], "build_reason": %d}`, selected.Name, model.BuildReasonFlagTriggerCLI))
+				resp, err := http.Post(fmt.Sprintf("http://%s/api/%s", h.webURL.Host, "trigger"), "application/json", bytes.NewBuffer(payload))
```

Comment:
Note: I tried to use `dispatch()` passing this Action:

https://github.com/tilt-dev/tilt/blob/688f5dfb982ce68de84977dc26e3d12b7f486355/internal/hud/server/actions.go#L7-L10

Like how the server handles it:

https://github.com/tilt-dev/tilt/blob/688f5dfb982ce68de84977dc26e3d12b7f486355/internal/hud/server/server.go#L270

But I encountered `import cycle not allowed`, so I ended up following the example in the linked issue here - https://github.com/tilt-dev/tilt/issues/2894#issuecomment-585866835 to just send a POST